docs/tutorial.md

```diff
@@ -0,0 +1,202 @@
+
+# finally create a small radioactive source
+source_s = pg4.geant4.solid.Tubs("Source_s", 0, 1, 1, 0, 2*pi, registry=reg)
+source_l = pg4.geant4.LogicalVolume(source_s, "G4_BRAIN_ICRP", "Source_L", registry=reg)
+pg4.geant4.PhysicalVolume([0, 0, 0], [0, 5, 0, "cm"], source_l, "Source", lar_l, registry=reg)
+```
+
+Note how we also created a small cylinder to represent a radioactive source
+later in the simulation.
+
+Now we can quickly visualize the result, still with *pyg4ometry*:
+
+```python
+# start an interactive VTK viewer instance
+viewer = pg4.visualisation.VtkViewerColoured(materialVisOptions={"G4_lAr": [0, 0, 1, 0.1]})
+viewer.addLogicalVolume(reg.getWorldVolume())
+viewer.view()
+```
+
+![Geometry visualization](img/tutorial-pyg4-view.jpg)
+
+By following instructions in the <project:./install.rst> section, you should
+have access to the `remage` executable. We are now ready to simulate some
+particle physics with it.
+
+Like any other Geant4-based application, we need to configure the simulation
+with a macro file. Standard Geant4 commands as well as custom commands (see the
+<project:./rmg-commands.rst>) are available.
+
+At the beginning of the file, we have to register the "sensitive detectors" (in
+our simple case, the two HPGes and the LAr). *remage* offers several types of
+predefined detectors, targeting different physical quantities of the particles
+that interact with them. HPGes are of type `Germanium`, while the LAr is of
+type `scintillator`. Their difference in terms of simulation output will be
+clear later, while inspecting it. As per specification of the
+`/RMG/Geometry/RegisterDetector` command, we need to provide a unique numeric
+identifier that will be used to label the detector data in the simulation
+output:
+
+```text
+/RMG/Geometry/RegisterDetector Germanium BEGe 001
+/RMG/Geometry/RegisterDetector Germanium Coax 002
+/RMG/Geometry/RegisterDetector Scintillator LAr 003
+```
+
+Now we can initialize the simulation. Additionally, let's setup some Geant4
+visualization to look at the tracks:
+
+```text
+/run/initialize
+
+# create a scene
+/vis/open OI
+/vis/scene/create
+/vis/sceneHandler/attach
+
+# draw the geometry
+/vis/drawVolume
+
+# setup better colors
+/vis/viewer/set/defaultColour black
+/vis/viewer/set/background white
+
+# and also show trajectories and particle hits
+/vis/scene/add/trajectories smooth
+/vis/scene/add/hits
+/vis/scene/endOfEventAction accumulate
+```
+
+Now with the actual physics. We want to start ten 1 MeV gammas from the
+radioactive source:
+
+```text
+/RMG/Generator/Confine Volume
+/RMG/Generator/Confinement/Physical/AddVolume Source
+
+/RMG/Generator/Select GPS
+/gps/particle gamma
+/gps/ang/type iso
+/gps/energy 1000 keV
+
+/run/beamOn 50
+```
+
+We can finally pass the GDML geometry and the macro to the `remage` executable
+and look at the result!
+
+```console
+$ remage --interactive --gdml-files geometry.gdml vis-gammas.mac
```

Comment:
maybe add the option to select an output file? (also below in the note block). Or do you plan to have this in the next (not yet written) part?